### ISSUE TITLE
aria-sort updates are not announced in Chrome

### ISSUE THREAD

**Issue by pablonete** [nvaccess/nvda#10774]
### Steps to reproduce:
Updating `aria-sort` programmatically is announced in Firefox but not in Chrome.
Update is done in response to user key stroke.

#### NVDA version:
2019.3 (installed using Windows setup)

**Comment by Adriani90** [comment]
@pablonete could you please provide a minimal test case or an example website where we can reproduce this?

**Comment by Adriani90** [comment]
Actually, closing as duplicate of #10890 which also contains an example.

**Comment by jcsteh** [comment]
Firefox does notify of aria-sort changes, so I've adjusted #10890 to cover the issue with NVDA not reporting aria-sort when the change occurs on something other than the focused element. However, Chrome doesn't notify of aria-sort changes at all, so it doesn't even work on the focused element. Thus, I'm reopening this.

STR:

1. Open this URL:
    `data:text/html,before<table><tr><th tabindex="0" aria-sort="ascending" onkeydown="if (event.key == ' ') this.setAttribute('aria-sort', 'descending');">Test</th><td>Data</td></tr></table>`
2. Control+home to top of document.
3. Press tab. NVDA should switch to focus mode.
4. press space.
    - Expected: NVDA should say "sorted descending"
    - Actual: It doesn't.

Works as expected in Firefox. Chrome needs to fire IA2_EVENT_OBJECT_ATTRIBUTE_CHANGED.

**Comment by aleventhal** [comment]
Filed https://bugs.chromium.org/p/chromium/issues/detail?id=1074380
It's an easy fix and should make it into Chrome 84.

**Comment by aleventhal** [comment]
The change has landed and should be in tomorrow's Canary.

**Comment by ObjectInSpace** [comment]
confirm this works in my canary (84.0.4141.0 but the fix is likely in an earlier version)

**Comment by Adriani90** [comment]
Thank you very much for your work on this. very appreciated.

Closing as works for me.
